packages/framework-integration-tests/integration/fixtures/cart-demo/package.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
I have mixed feelings with this fixture. On one hand it its great that it checks all the dependencies and the versions. On the other hand, it will cause many integration failures since dependencies will be update often, as well as the `@boostercloud` packages on every release. Is there any way that we could compare this one using regex?

Reply:
That's an excellent question. Thanks for pointing that out, I am going to think about it. I agree with you.

Reply:
Well, at least this test will fail if things change...

Reply:
I was thinking about solving this issue using regex but I haven't found a smooth approach to do that without maintaining a complex regex. Instead, I just created a new method where I check that the key-values from the generated package.json are the same from the expected one, but the `dependencies` and `devDependencies` section where I only check the keys and not the values. Here is the commit f3d2eea4a6e1b4a36498e3c08fd11f5a24e896ba

Reply:
Take into the account that is the generated package.json has an invalid version of a library for any reason (let's say a typo in the Booster templates), these tests will fail for that reason anyway, so I think is not worth trying to check whether or not a version meets a correct npm version pattern.

Reply:
sounds good to me. That just made me find out that we should probably not ally * for the framework-aws-provider